src/xdocs/checks/annotation/missingoverride.xml

```diff
@@ -67,86 +67,103 @@
         </div>
       </subsection>
       <subsection name="Examples" id="Examples">
-        <p> To configure the check:</p>
+        <p id="Example1-config"> To configure the check:</p>
         <source>
-&lt;module name=&quot;MissingOverride&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;MissingOverride&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>Example:</p>
+        <p id="Example1-code">Example:</p>
         <source>
-class Test extends SuperClass {
+class Example1{
+  public void test1(){}
```

Comment:
please name `class A` to be `class Example1` and we do not need SuperClass to have same name and NOT  visible to user.